cie_infra/__manifest__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "cie_infra",
+
+    'summary': """
+        this module allows to visualize our servers throught an inventary. """,
+
+    'description': """
+        The purpose of this module is to visualize all the information 
+	contains on our server. It means the datacenter, the location,
+	the different instance on it and finally the different module.
+    """,
+
+    'author': "CoopItEasy/EliotVanGoethem",
```

Comment:
Can you put only `Coop IT Easy` here. Your name will be put in the Readme of the module in the Author section (we will see this later).